frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -22,6 +23,27 @@ const applyPods = (podsData: PodRingData, dc: PodRCData) => {
   return podsData;
 };
 
+export const podRingLabel = (resourceObj: K8sResourceKind, isScalingAllowed: boolean) => {
+  const title =
+    resourceObj.status.availableReplicas || (isScalingAllowed ? 'Scaled to 0' : 'Autoscaled');
+
+  const subTitle =
```

Comment:
The boolean logic here is overly complex. We are sacrificing readability to save a few lines of code. We should use a few more variables or functions to make this a bit more readable. We try to avoid nested ternaries where possible.